src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/cli/usecases/preferences/SettingsCLIPrompt.java

```diff
@@ -27,6 +27,11 @@ public void start() {
   private BaseCLIPrompt showActionMenuPrompt() {
     printDefault("Which setting do you want to change?");
     Map<String, BaseCLIPrompt> promptMap = new LinkedHashMap<>();
+
+    promptMap.put("Change password",
+        new ChangePasswordCLIPrompt(this, collectionSettings.changePasswordUseCase(),
+            collectionSettings.authSessionUseCase()));
+
```

Comment:
```suggestion
    promptMap.put("Change password",
        new ChangePasswordCLIPrompt(this, collectionSettings.changePasswordUseCase());

```

this also has to be changed accordingly